server/graphql/dog-licenses.js

```diff
@@ -8,40 +7,63 @@ import type {
 
 export const Schema = `
 type DogLicense {
-  id: String!,
-  firstName: String!,
-  lastName: String!,
-  deathDate: String,
-  deathYear: String!,
-  pending: Boolean,
-  age: String,
-}
-
-# Pages are 1-indexed to make the UI look better
-type DogLicenseSearch {
-  page: Int!,
-  pageSize: Int!,
-  pageCount: Int!,
-  results: [DogLicense!]!,
-  resultCount: Int!,
+  firstName: String,
```

Comment:
This looks great! As you develop, you may want to revisit this and add `!` after the types (_e.g._ `String!`) for anything that’s not going to ever be null (like `id`). As the Flow types make their way to your JS client code, that will allow you to avoid unnecessary nullability checks.